DeviceAdministration/Infrastructure.UnitTests/Infrastructure/SecurityKeyGeneratorTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.BusinessLogic;
+using Xunit;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.DeviceAdmin.Infrastructure.UnitTests
+{
+    public class SecurityKeyGeneratorTests
+    {
+        [Fact]
+        public void CreateRandomKeysTest()
+        {
+            var securityKeyGenerator = new SecurityKeyGenerator();
+            var keys = securityKeyGenerator.CreateRandomKeys();
+
+            Assert.NotNull(keys);
+            Assert.NotEqual(keys.PrimaryKey, keys.SecondaryKey);
+        }
```

Comment:
Can we have a second assertion that tests against a second pair of keys to make sure this isn't deterministic?